packages/polling-controller/package.json

```diff
@@ -61,7 +61,7 @@
     "typescript": "~4.9.5"
   },
   "peerDependencies": {
-    "@metamask/network-controller": "^18.1.2"
+    "@metamask/network-controller": "^18.0.0"
```

Comment:
I think this one should be aligned with the `dependencies` field? So if dropping here, then we'd also like to drop to `dependencies["@metamask/network-controller"] = "^18.1.0"` ?